provisioning/roles/nagios/tasks/main.yml

```diff
@@ -6,15 +6,11 @@
       - nagios
       - nagios-plugins-all
       - nagios-plugins-nrpe
+      - nsca
       - nrpe
       - httpd
   become: true
 
-- name: install older nsca 2.6.2
```

Comment:
NSCA-2.6.2 for el5 has dependency conflicts with the el7 nagios. 

Why do we use this old version? (Current is 2.9.x)

Reply:
I am not certain, my guesses:
- there are issues with the clients using an old version of the client
- perhaps this was related to the nsca process getting out of control (spawning to many times), bringing the server down?

Ah, I was going to point to this commit where that nsca 2.6.2 was chosen over the yum version:
https://github.com/HiSPARC/publicdb/commit/107640eb3fb0593822254a7387ab8244ba93d7fd

But the parent commit has a more interesting commit message:
https://github.com/HiSPARC/publicdb/commit/5229a471fc550eb3b76ae5baee36622c08809442

Reply:
ATM most of the stations do not have a functional VPN. And fixing the VPN means upgrading the entire installer to the most recent. With new certificates/openvpn/nagios.

So that might fix this. I'm kind of hoping we can switch to the EPEL ncsa... (Fingers crossed)

Reply:
5229a47 might be the most interesting (scariest) commit in this repo.

Reply:
😂 indeed, scary, and a bit difficult to track down.